src/main/java/xyz/nucleoid/plasmid/game/player/PlayerSet.java

```diff
@@ -100,6 +100,11 @@ default void sendTitle(Text title, Text subtitle, int fadeInTicks, int stayTicks
         this.sendPacket(new TitleS2CPacket(TitleS2CPacket.Action.SUBTITLE, subtitle));
     }
 
+    default void sendActionbar(Text text, int fadeInTicks, int stayTicks, int fadeOutTicks) {
```

Comment:
The underlying method for this should be in `PlayerOps`

Reply:
Done